benefits/core/migrations/0002_sample_data.py

```diff
@@ -95,35 +95,19 @@ def load_sample_data(app, *args, **kwargs):
     EligibilityVerifier = app.get_model("core", "EligibilityVerifier")
 
     verifier1 = EligibilityVerifier.objects.create(
```

Comment:
c85f4d2c5d930572f70a6b8ac1f19d232fa3b906 is the critical commit of the PR - I removed the _extra_ sample verifier we had (previously standing in as a placeholder for the DMV), and moved the existing sample OAuth verifier up and renamed it to `verifier1`.

This had a big impact on the PO files